tests/test_level/test_level_01.py

```diff
@@ -545,9 +545,21 @@ def test_lines_may_end_in_spaces(self):
     #
     # negative tests
     #
-    def test_lines_with_space_gives_invalid(self):
-        code = " print Hallo welkom bij Hedy! "
-        self.single_level_tester(code=code, exception=hedy.exceptions.InvalidSpaceException)
+    def test_print_with_space_gives_invalid(self):
+        code = " print Hallo welkom bij Hedy!"
+
+        self.multi_level_tester(
+            code=code,
+            exception=hedy.exceptions.InvalidSpaceException,
+            max_level=3)
+
+    def test_ask_with_space_gives_invalid(self):
+        code = " ask Hallo welkom bij Hedy?"
+
+        self.multi_level_tester(
+            code=code,
+            exception=hedy.exceptions.InvalidSpaceException,
+            max_level=1)
 
     def test_lines_with_spaces_gives_invalid(self):
         code = " print Hallo welkom bij Hedy!\n print Hallo welkom bij Hedy!"
```

Comment:
A single line that starts with a space is allowed (above), but multiple lines still give an error? Is that by design? Or is it just me misinterpreting things?

Reply:
No this too will give a (desired) exception. I made the tests look more similar to make that more clear.

Reply:
I guess I wasn't focused enough after the previous review.

Reply:
No problem @arjank!! A lot has changed in the grammar/parser and test suite so I totally understand that this was hard to understand, plus... your comment made me realize the tests were not similar enough.